### PR TITLE
initial podman mods

### PR DESCRIPTION
**Pull request by tchughesiv**
this still requires some podman related content changes... but all of the commands should be functional through ch5, except the atomic piece which doesn't seem compatible w/ podman.  There's also still work to do on the bonus chapter.

### CODE REVIEW

File: labs/lab1/chapter1.md [hunk 1]
```diff
@@ -34,40 +34,40 @@ Now check on the status of docker.
 $ systemctl status docker
 ```
```
Comment (dustymabe) [inline labs/lab1/chapter1.md]: we talk about podman in the words, but all of these commands are referencing docker

File: labs/lab1/chapter1.md [hunk 2]
```diff
@@ -34,40 +34,40 @@ Now check on the status of docker.
...
 
-Run `docker {help,info}` to check out the storage configuration and how to find more information. 
+Run `podman {help,info}` to check out the storage configuration and how to find more information. 
 ```bash
-$ docker --help
-$ docker run --help
-$ docker info
+$ sudo podman --help
+$ sudo podman run --help
```
Comment (dustymabe) [inline labs/lab1/chapter1.md]: let's take the sudo off these first two commands. we don't need root access to run `--help`

File: labs/lab2/chapter2.md [hunk 3]
```diff
@@ -26,46 +26,44 @@ Our monolithic application we are going to use in this lab is a simple wordpress
...
-$ docker ps
+$ sudo podman run -p 80 --name=bigapp -e DBUSER=user -e DBPASS=mypassword -e DBNAME=mydb -d bigimg
+$ sudo podman ps
 ```
 
-Take a look at some of the arguments we are passing to docker.  We are telling docker that the image will be listening on port 80 inside the container and to randomly assign a port on the host that maps to port 80 in the container. Next we are providing a ```name``` of ```bigapp```. After that we are setting some environment variables that will be passed into the container and consumed by the configuration scripts to set up the container. Finally, we pass it the name of the image that we built in the prior step.
+Take a look at some of the arguments we are passing to docker.  We are telling podman that the image will be listening on port 80 inside the container and to randomly assign a port on the host that maps to port 80 in the container. Next we are providing a ```name``` of ```bigapp```. After that we are setting some environment variables that will be passed into the container and consumed by the configuration scripts to set up the container. Finally, we pass it the name of the image that we built in the prior step.
```
Comment (dustymabe) [inline labs/lab2/chapter2.md]: s/docker/podman

File: labs/lab2/chapter2.md [hunk 4]
```diff
@@ -26,46 +26,44 @@ Our monolithic application we are going to use in this lab is a simple wordpress
...
 
 ## Exploring the Running Container
 
-Now that the container is running we will explore the container to see what's going on inside. First off, the processes were started and any output that goes to stdout will come to the console of the container. You can run `docker logs` to see the output. To follow 
+Now that the container is running we will explore the container to see what's going on inside. First off, the processes were started and any output that goes to stdout will come to the console of the container. You can run `podman logs` to see the output. To follow 
 or "tail" the logs use the `-f` option.
 
 **__NOTE:__** You are able to use the **name** of the container rather than the container id for most `docker` commands.
```
Comment (dustymabe) [inline labs/lab2/chapter2.md]: s/docker/podman

File: labs/lab3/chapter3.md [hunk 5]
```diff
@@ -176,123 +176,102 @@ Save the Dockerfile and exit the editor.
...
-$ docker exec $(docker ps -ql) stat --format="%U" /var/lib/mysql
-$ docker logs $(docker ps -ql)
-$ docker ps
-$ curl localhost:3306
+$ sudo podman exec wordpress stat --format="%U" /var/log/httpd/access_log
+$ sudo podman logs wordpress
+$ sudo podman ps
 ```
 
   **Note**: the `curl` command does not return useful information but demonstrates
```
Comment (dustymabe) [inline labs/lab3/chapter3.md]: looks like the curl command got removed.. was that intentional?

File: labs/lab3/chapter3.md [hunk 6]
```diff
@@ -176,123 +176,102 @@ Save the Dockerfile and exit the editor.
...
 
-1. Create the local directories for persistent storage & set permissions for container runtime.
+1. Create the local directories for persistent storage.
 
-        $ sudo mkdir -p /var/lib/mariadb /var/lib/wp_uploads
-        $ sudo chown 27 /var/lib/mariadb /var/lib/wp_uploads
+        $ mkdir -p ~/workspace/logs ~/workspace/mysql ~/workspace/run ~/workspace/wp_logs
 
-1. Run the database image to confirm connectivity. It takes some time to discover all of the necessary `docker run` options.
+1. Run the wordpress image first. It takes some time to discover all of the necessary `podman run` options.
```
Comment (dustymabe) [inline labs/lab3/chapter3.md]: is there a reason we changed the order and now start wordpress first? I just ran through this and it didn't seem to work. investigating now

Comment (dustymabe) [reply labs/lab3/chapter3.md]: ok. this was a copy/paste error on my part. I'd still like to know if there was a technical reason why we changed the order. I think it's more natural for a user/admin to bring up a database before they bring up their app.

File: labs/lab3/chapter3.md [hunk 7]
```diff
@@ -176,123 +176,102 @@ Save the Dockerfile and exit the editor.
...
-$ curl localhost:3306
+$ sudo podman exec wordpress stat --format="%U" /var/log/httpd/access_log
+$ sudo podman logs wordpress
+$ sudo podman ps
 ```
 
   **Note**: the `curl` command does not return useful information but demonstrates
             a response on the port.
 
 5. Test the Wordpress image to confirm connectivity. Additional run options:
```
Comment (dustymabe) [inline labs/lab3/chapter3.md]: I don't know if this is the right text any longer

File: labs/lab3/chapter3.md [hunk 8]
```diff
@@ -176,123 +176,102 @@ Save the Dockerfile and exit the editor.
...
+$ sudo podman exec mariadb stat --format="%U" /var/log/mariadb/mariadb.log
+$ sudo podman logs mariadb
+$ sudo podman ps
+$ sudo podman exec mariadb curl localhost:3306
 $ curl -L http://localhost:8080
 ```
 
 You may also load the Wordpress application in a browser to test its full functionality @ `http://<YOUR AWS VM PUBLIC DNS NAME HERE>:8080`.
 
 ### Simplify running containers with the atomic CLI
```
Comment (dustymabe) [inline labs/lab3/chapter3.md]: i think we should possibly nuke this section. atomic CLI doesn't seem to work with podman and buildah [doesn't yet support caching build layers for `buildah bud`](https://github.com/projectatomic/buildah/issues/601) so we highlight the fact that changing one line in the Dockerfile requires it to be rebuilt from scratch.

**Comment by tchughesiv** [comment]
It’s a technical reason ... it’s got to do with the way linking networks currently functions in podman. This was the best I was able to do in order to still support chapter 4’s DB stop and start demonstration.

File: labs/lab4/chapter4.md [hunk 9]
```diff
@@ -60,7 +61,7 @@ And.. Your site is back! Fortunately wordpress seems to be designed such that it
 
 Finally, let's kill off these containers to prepare for the next section.
 ```bash
-$ docker rm -f wordpress mariadb
+$ sudo podman rm -f wordpress mariadb
```
Comment (dustymabe) [inline labs/lab4/chapter4.md]: i think the order on this needs to be flipped (mariadb and then wordpres). I hit an error and I think it's related.

**Comment by whitel** [comment]
Merging w/ unresolved issues because I will attempt to resolve them in a new PR

File: labs/lab1/chapter1.md [hunk 10]
```diff
@@ -34,40 +34,40 @@ Now check on the status of docker.
...
+$ sudo podman info
 ```
 
-Take a look at the docker images on the system. You should see some Openshift images that are cached.
+Take a look at the podman images on the system. You should see some Openshift images that are cached.
```
Comment (dustymabe) [inline labs/lab1/chapter1.md]: we don't see any images in this step because we haven't used podman yet

**Pull request merged**